PyAlbany/examples/paper/.gitignore

```diff
@@ -1,4 +1,5 @@
 steady2d*
 *.jpeg
 *.sh
-*.txt
\ No newline at end of file
+*.txt
+input_scalar_updated.yaml
```

Comment:
why did you add `input_scalar_updated.yaml` to `.gitignore`?

Reply:
Because I used an input file `PyAlbany/examples/paper/input_scalar_to_update.yaml` as a "template" to generate an updated one (updated in the script) and then write it to the disc to use as input argument of the executable. We don't want to commit the updated one as it changes all the time.

This is in the context of the MC simulation using the C++ executable.

Reply:
I see. It makes sense.